File: example-files/homoglyph.js
const [ ENV_PROD, ENV_DEV ] = [ 'PRODUCTION', 'DEVELOPMENT'];
const environment = '...';
function isUserAdmin(user) {
   if(environmentǃ=ENV_PROD){
       // bypass authZ checks in DEV
       return true;
   }
   return false;
}

// Fine in comment ǃ
"Fine in strings ǃ"

File: example-files/homoglyph.go
package main

const EnvProd, EnvDev = "PRODUCTION", "DEVELOPMENT"

func check() bool {
	environment := "..."
	if environmentǃ= EnvProd {
		// bypass authZ checks in DEV
		return true
	}
	return false
}

// Fine in comment ǃ
const str = "Fine in strings ǃ"
const rstr = `Fine in raw strings ǃ`
